scania-digital-design-system/sdds
# feat(dropdown): add openDirection prop

**Describe pull-request**  
Added a prop that allows the user to specify which direction a dropdown should be opened. This prop is be default set to auto, which uses our current solution of calculating which way is the correct one. This solution is implemented for both old and new package. 

**Solving issue**  
Fixes: [AB#2740](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2740)

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Dropdown
3. Check the "Open direction" control and try it out.
4. Check out the branch locally and run npm start in the components directory.
5. Check in Components -> Dropdown
6. Check the "Open direction" control and try it out.


**Screenshots**  
-

**Additional context**  
-
